stl/inc/xhash

```diff
@@ -269,6 +269,12 @@ struct _Hash_vec {
             _Aliter_traits::max_size(_Mypair._Get_first()));
     }
 
+    _NODISCARD size_type capacity() const noexcept {

```

Comment:
No change requested: This `_Hash_vec` is already using `vector`'s non-`_Ugly` names `size` and `max_size`, so `capacity` is fine (macroization and other collisions aren't possible here). This is locally consistent and a special exception to our usual conventions.